spec/system/admin/invoice_print_spec.rb

```diff
@@ -5,7 +5,7 @@
 describe '
     As an administrator
     I want to print a invoice as PDF
-', js: false do
+', type: :feature, js: false do
```

Comment:
I think this is the way to go :+1: 

I've tried locally to keep `:cuprite` as the default driver (basically undoing any changes to our `cuprite_setup.rb`) as removing this brings back the default capybara configuration (i.e., `:selenium` as a driver) and set `:rack_test` only for this file with:

`Capybara.current_driver = :rack_test`

following this here:
https://github.com/teamcapybara/capybara#selecting-the-driver

I've done this in this commit d33e808c249ddcfedafaf0778c7bc80eb9a8b870, but somehow I could not push into your branch, and ended up creating another one - here: https://github.com/openfoodfoundation/openfoodnetwork/pull/9577

Reply:
Ah, cool, that's great, too. I guess, the big decision is: Do we want to use rack test for all non-js specs for do we want to use Chrome (cuprite) for all specs except where we need to download PDFs?

* Rack test is a lot faster.
* All specs to be run with cuprite would need the javascript flag again.
* Testing with Chrome is more realistic and we can debug more easily with a real browser.
* At some point in the future, we may find a way to download PDFs with Chrome as well.

So I think that it comes down to speed vs accuracy. Either way I'm confident that we can make the config work.

I'm leaning towards rack test for speed where possible.

Reply:
> I'm leaning towards rack test for speed where possible.

Sounds good. Maybe a good starting point would be to run/convert the 3 slowest (system) on rack tests, and re-evaluate in terms of accuracy loss.

Reply:
Updated the [issues on this epic](https://github.com/openfoodfoundation/openfoodnetwork/issues/8047), this changed a bit since we've migrated everything into system specs.

Reply:
> Maybe a good starting point would be to run/convert the 3 slowest (system) on rack tests

I doubt that this will work. Rack tests don't support Javascript and our slowest tests are probably around the checkout which needs Javascript (at least for Stripe payments). Checkout is also very important to be more realistic. Maybe the Split checkout will work without JS though?

Reply:
> Do we want to use rack test for all non-js specs for do we want to use Chrome (cuprite) for all specs except where we need to download PDFs?

Right, I see your point now - sounds like a good idea to use `rack_test` for all non-js specs - not the slowest performing specs, those surely require js. I understand your approach on this PR here as well. It might be a bit of an effort to make this migration, but surely worth it.

Other than:
> Testing with Chrome is more realistic and we can debug more easily with a real browser.

There is really no downside to it.